src/RedisSubscriber.ts

```diff
@@ -83,9 +81,10 @@ export default class RedisSubscriber
     let iteratorPromise = queue.iterator;
     if (parseMessage) {
       // Workaround for Flow.
-      const parseMessageFn: string => any = parseMessage;
+      const parseMessageFn: (msg: string) => any = parseMessage;
       iteratorPromise = iteratorPromise.then(iterator =>
-        map(iterator, parseMessageFn),
+        // FIXME: type this
+        map(iterator as any, parseMessageFn),
```

Comment:
Same here with the `AsyncIterable` vs `AsyncIterator`

Reply:
should be async iterables everywhere